src/libraries/System.Drawing.Common/src/System.Drawing.Common.Forwards.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// This is required for back-compatibility with legacy Xamarin which had these types in System.Drawing.Common.dll
```

Comment:
You are adding these to the implementation assembly only. That's intentional, right?

Reply:
I think that is intentional. We only need the type-forwarders for binary compatibility at runtime. I am not sure how it works with the ApiCompat checks though.

`SystemColors` and `ColorTranslator` have their respective `TypeForwardedTo` in the ref sources as well. These two types were moved between assemblies between .NET Core 2 and 3 IIRC. Do I understand correctly that the implementation assembly previously got the `TypeForwardedTo` for these two types thanks to `IsPartialFacadeAssembly = true`?

Reply:
Actually, we'd still need the IsPartialFacadeAssembly for the other TFMs so we'd need a conditional. I'll just leave it as is.

Reply:
I'm fine with that approach and it follows what we do in our shims builds (manual type forwards only exposed in src but not ref). Just to get a second pair of eyes, let me cc'ing @ericstj